1.10/Dockerfile

```
@@ -7,13 +7,13 @@ RUN set -eux ; \
     coreutils \
     git \
     make \
-    mercurial \
     openssh-client \
     patch \
     subversion \
     tini \
     unzip \
-    zip
+    zip \
+    [ $(apk --print-arch) != "x86" ] && echo mercurial
```

I think you're missing some `$()` here, maybe?

```suggestion
    zip \
    $([ "$(apk --print-arch)" != "x86" ] && echo mercurial)
```

(hence the build failure :smile:)

---

Yeah it took a few iterations to figure out; my bash-fu is dusty. Forgot to put the final fix in both files :sweat_smile: